CactbotOverlay/FFXIVProcess.cs

```diff
@@ -368,6 +371,63 @@ public enum EntityJob {
       return active_process_id == process_.Id;
     }
 
+    public class StatusEffect {
+      private static Dictionary<int, float> maxDurations = new Dictionary<int, float>();
+      public ushort id = 0;
+      public byte stackCount = 0;
+      public float duration = 0;
+      public uint casterId = 0;
+      public float maxDuration = 0;
+
+      public static int offsetId = 0x0;
+      public static int offsetStackCount = 0x2;
+      public static int offsetDuration = 0x4;
```

Comment:
Sorry that the C# style is weird, but most of the C# code uses hacker_style_underscores and not camelCase.  The only time it uses camelCase is when it's for JSEvent structs that are serialized to Javascript, so that those can be in the right style.  It'd be nice if this could all be consistent.